### PR TITLE
add limit add docs

### PR DESCRIPTION
**Pull request by timchenxiaoyu**
add limit add docs

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6975)

### CODE REVIEW

File: docs/tasks/configure-pod-container/quality-service-pod.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ kubectl delete pod qos-demo --namespace=qos-example
 A Pod is given a QoS class of Burstable if:
 
 * The Pod does not meet the criteria for QoS class Guaranteed.
-* At least one Container in the Pod has a memory or cpu request.
+* At least one Container in the Pod has a memory or cpu request or limit.
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/configure-pod-container/quality-service-pod.md]: if the user only set limit not request, as a default result, the request value equal the limit value, and that is `guaranteed`.

**Comment by zhangxiaoyu-zidif** [comment]
>For a Pod to be given a QoS class of Guaranteed:
Every Container in the Pod must have a memory limit and a memory request, and they must be the same.
Every Container in the Pod must have a cpu limit and a cpu request, and they must be the same.

**Comment by zhangxiaoyu-zidif** [comment]
please feel free to contact me if any question. thanks for your contribution.

**Comment by timchenxiaoyu** [comment]
I change Burstable definition， not Guaranteed？

**Pull request closed without merging**